gemdaq-testing/gemsupervisor/src/common/GEMGLIBSupervisorWeb.cc

```diff
@@ -411,7 +411,7 @@ void gem::supervisor::GEMGLIBSupervisorWeb::webTrigger(xgi::Input * in, xgi::Out
   hw_semaphore_.take();
 
   INFO(" webTrigger: sending L1A");
-  optohybridDevice_->SendL1A(1);
+  optohybridDevice_->SendL1A(100);
```

Comment:
Please do not commit this change to the central pull request
In the "official" version, it should be SendL1A(1) such that it does exactly what the log message states